nixos/modules/services/x11/desktop-managers/gnome.nix

```diff
@@ -351,7 +351,7 @@ in
 
     (mkIf serviceCfg.core-os-services.enable {
       hardware.bluetooth.enable = mkDefault true;
-      hardware.pulseaudio.enable = mkDefault true;
+      hardware.pulseaudio.enable = mkDefault false; # be pipewire by default
```

Comment:
Why not just remove this?

Reply:
Good point, I hadn't checked the default default value :)